docs/CODE_OF_CONDUCT-km.md

```diff
@@ -0,0 +1,47 @@
+# ក្រមសីលធម៌របស់អ្នកចូលរួម
+
+ក្នុងនាមជាអ្នករួមចំណែក និងអ្នកថែរក្សាគម្រោងនេះ និងក្នុងគោលបំណងជំរុញសហគមន៍បើកចំហ និងស្វាគមន៍ 
+យើងសន្យាគោរពមនុស្សទាំងអស់ដែលរួមចំណែកតាមរយៈការរាយការណ៍បញ្ហា ការបង្ហោះសំណើលក្ខណៈពិសេស 
+ការធ្វើបច្ចុប្បន្នភាពឯកសារ ការដាក់សំណើរទាញ ឬបំណះ និងសកម្មភាពផ្សេងទៀត។
+
+យើងប្តេជ្ញាធ្វើឱ្យការចូលរួមនៅក្នុងគម្រោងនេះដោយគ្មានការយាយី
+បទពិសោធន៍សម្រាប់មនុស្សគ្រប់គ្នា ដោយមិនគិតពីកម្រិតនៃបទពិសោធន៍ ភេទ ភេទ
+អត្តសញ្ញាណ និងការបញ្ចេញមតិ ទំនោរផ្លូវភេទ ពិការភាព រូបរាងផ្ទាល់ខ្លួន
+ទំហំរាងកាយ ជាតិសាសន៍ ជាតិសាសន៍ អាយុ សាសនា ឬសញ្ជាតិ។
+
+ឧទាហរណ៍នៃអាកប្បកិរិយាមិនអាចទទួលយកបានដោយអ្នកចូលរួមរួមមាន:
+
+* ការប្រើប្រាស់ភាសាផ្លូវភេទ ឬរូបភាព
+* ការវាយប្រហារផ្ទាល់ខ្លួន
+* តិះដៀល ឬ ជេរប្រមាថ/ មតិប្រមាថ
+* ការបៀតបៀនសាធារណៈ ឬឯកជន
+* ការផ្សព្វផ្សាយព័ត៌មានឯកជនរបស់អ្នកដទៃ ដូចជារូបវន្ត ឬអេឡិចត្រូនិច អាសយដ្ឋាន ដោយគ្មានការអនុញ្ញាតច្បាស់លាស់
+* អាកប្បកិរិយាអសីលធម៌ ឬគ្មានវិជ្ជាជីវៈផ្សេងទៀត។
+
+អ្នកថែទាំគម្រោងមានសិទ្ធិ និងការទទួលខុសត្រូវក្នុងការដកចេញ កែសម្រួល ឬ
+បដិសេធមតិយោបល់ ការប្តេជ្ញាចិត្ត កូដ ការកែសម្រួលវិគី បញ្ហា និងការរួមចំណែកផ្សេងទៀត។
+ដែលមិនស្របតាមក្រមសីលធម៌នេះ ឬហាមឃាត់ជាបណ្តោះអាសន្ន ឬ
+អ្នករួមចំណែកអចិន្ត្រៃយ៍សម្រាប់អាកប្បកិរិយាផ្សេងទៀតដែលពួកគេចាត់ទុកថាមិនសមរម្យ
+គំរាមកំហែង វាយលុក ឬបង្កគ្រោះថ្នាក់។
+
+តាមរយៈការអនុម័តក្រមសីលធម៌នេះ អ្នកថែទាំគម្រោងបានប្តេជ្ញាចិត្តចំពោះខ្លួនឯង
+ការអនុវត្តគោលការណ៍ទាំងនេះដោយយុត្តិធម៌ និងជាប់លាប់ចំពោះគ្រប់ទិដ្ឋភាពនៃការគ្រប់គ្រង
+គម្រោងនេះ។ អ្នកថែទាំគម្រោងដែលមិនអនុវត្តតាម ឬអនុវត្តក្រមនៃ
+ការប្រព្រឹត្តអាចនឹងត្រូវដកចេញជាអចិន្ត្រៃយ៍ពីក្រុមគម្រោង។
+
+ក្រមសីលធម៌នេះអនុវត្តទាំងក្នុងចន្លោះគម្រោង និងក្នុងទីសាធារណៈ
+នៅពេលដែលបុគ្គលម្នាក់តំណាងឱ្យគម្រោង ឬសហគមន៍របស់ខ្លួន។
+
+ករណីនៃការបំពាន ការយាយី ឬអាកប្បកិរិយាមិនអាចទទួលយកបានអាចកើតមាន
+រាយការណ៍ដោយទាក់ទងអ្នកថែទាំគម្រោងនៅ victorfelder នៅ gmail.com ។ ទាំងអស់។
```

Comment:
Anyway, here I think it is a gmail address right? 
I saw in https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CODE_OF_CONDUCT.md
is using `victorfelder at gmail.com` as well. 
Would look betting if we just use a proper email address instead? `victorfelder@gmail.com`
It is just my idea. You may have your own reason.